source/assets/scripts/main.js

```diff
@@ -0,0 +1,101 @@
+// main.js
+
+const $ = (selector) => document.querySelector(selector);
+
+
+const featureRecipes = [
+  'https://introweb.tech/assets/json/ghostCookies.json',
+  'https://introweb.tech/assets/json/birthdayCake.json',
+  'https://introweb.tech/assets/json/chocolateChip.json',
+  'assets/recipes/meatball.json',
+  'assets/recipes/quickEasyPaste.json',
+  'assets/recipes/pumpkinMuffins.json'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group16/cse110-fa21-group16/pullRequest?prid=8393580)
```suggestion
  "assets/recipes/pumpkinMuffins.json"
```